test/mongoDbTest.js

```diff
@@ -6,26 +6,42 @@ const should = chai.should();
 
 chai.use(chaiHttp);
 
+const genreArr = [
+  'action',
+  'animation',
+  'comedy',
+  'documentary',
+  'drama',
+  'family',
+  'fantasy',
+  'international',
+  'horror',
+  'musical',
+  'mystery',
+  'romance',
+  'sci_fi',
+  'thriller',
+  'western',
+];
 const id = Math.floor(Math.random() * 300000);
+const genre = Math.floor(Math.random() * 15);
 
 describe('UserMovies', () => {
-
   it('should contain at least 1,000,000 users', () => {
```

Comment:
I'm wondering how you're tests are performing without passing in the done parameter (and invoking at the end of your async process). Your tests aren't timing out?

Reply:
I am not having any timeout issues.